.env.sample

```diff
@@ -75,3 +75,21 @@ TFM_API_KEY=TESTAPIKEY
 PORTAL_API_KEY=TESTAPIKEY
 DTFS_CENTRAL_API_KEY=TESTAPIKEY
 EXTERNAL_API_KEY=TESTAPIKEY
+
+# AZURE SSO LOGIN
+AZURE_SSO_TENANT_SUBDOMAIN=
+AZURE_SSO_REDIRECT_URI=
+AZURE_SSO_POST_LOGOUT_URI=
+AZURE_SSO_CLIENT_ID=
+AZURE_SSO_AUTHORITY=
+AZURE_SSO_CLIENT_SECRET=
+
+AZURE_SSO_GROUP_READ_ONLY=
+AZURE_SSO_GROUP_UNDERWRITERS=
+AZURE_SSO_GROUP_UNDERWRITING_SUPPORT=
+AZURE_SSO_GROUP_UNDERWRITER_MANAGERS=
```

Comment:
Should this be `AZURE_SSO_GROUP_UNDERWRITING_MANAGERS` ?

Reply:
Not sure, might be legacy or terminology thing.

UNDERWRITING_MANAGERS -> 7 mentions in code
UNDERWRITER_MANAGERS -> 115 mentions in code